Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -151,7 +151,7 @@
 #define ENABLE_APPLE_PAY_UPDATE_SHIPPING_METHODS_WHEN_CHANGING_LINE_ITEMS 1
 #endif
 
-#if !defined(ENABLE_APPLE_PAY_AMS_UI) && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 120300) || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 150400) || PLATFORM(VISION))
+#if !defined(ENABLE_APPLE_PAY_AMS_UI) && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 120300) || PLATFORM(IOS) || PLATFORM(VISION))
```

Comment:
I think macOS 12 is 2021 too

Reply:
12.3 is a point release of Monterey, which we still support.

Reply:
As discussed in Slack, no, we don't support SDKs older than 12.3.